commitlint.config.js

```diff
@@ -1 +1,131 @@
-module.exports = { extends: ["@commitlint/config-conventional"] };
+module.exports = {
```

Comment:
Would it be easier to leave config-conventional, and only override the rules we want to tweak?